h/celery.py

```diff
@@ -98,12 +98,6 @@ def bootstrap_worker(sender, **kwargs):
 
 
 @signals.task_prerun.connect
-def reset_feature_flags(sender, **kwargs):
```

Comment:
Exactly: we're covered by [these lines](https://github.com/hypothesis/h/blob/8da2ea3f4ede59a12b91d928c22357237a97bbd5/h/celery.py#L113-L122).